iotdb-core/datanode/src/test/java/org/apache/iotdb/db/protocol/rest/GrafanaApiServiceTest.java

```diff
@@ -359,4 +359,92 @@ public void variableTest() {
       fail(e.getMessage());
     }
   }
+
+  @Test
+  public void expressionWithAggGroupByTimeTest() {
+    CloseableHttpClient httpClient = HttpClientBuilder.create().build();
+    rightInsertTablet(httpClient);
+
+    CloseableHttpResponse response = null;
+    try {
+      HttpPost httpPost = getHttpPost("http://127.0.0.1:18080/grafana/v1/query/expression");
+      String sql =
+          "{\"expression\":[\"count(s4)\"],\"prefixPath\":[\"root.sg25\"],\"startTime\":1635232143960,\"endTime\":1635232153960,\"control\":\"group by([1635232143960,1635232153960),1s)\"}";
```

Comment:
Don't use magic string, use private static constant instead.

Reply:
Same as other tests. The test can contain magic characters.